pyrovision/models/rexnet.py

```diff
@@ -0,0 +1,153 @@
+# Copyright (C) 2021, Pyronear contributors.
+
+# This program is licensed under the GNU Affero General Public License version 3.
+# See LICENSE or go to <https://www.gnu.org/licenses/agpl-3.0.txt> for full license details.
+
+from holocron.models.rexnet import ReXNet
```

Comment:
You just need to import models, so that we can dynamically call the specific rexnet functions